spec/generator/scheme_spec.rb

```diff
@@ -1,44 +1,41 @@
-require 'spec_helper'
+require 'xcodeproj'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.